File: Circuit/digsig/adapter/msg_handler.py
import json
import base64

def objToU8(x):
    return [str(ord(char)) for char in x]


def read_message_from_json(json_file):
    with open(json_file, 'r') as f:
        obj = json.load(f)
        return json.dumps(obj, separators=(',', ':'))


with open("./Circuit/digsig/adapter/response.json", "r") as json_file:
    msg_og = json.load(json_file)

sig_og = msg_og["signature"]
del msg_og["signature"]
msg_og = json.dumps(msg_og, separators=(',', ':'))

msg = objToU8(msg_og)
sig_bytes = base64.b64decode(sig_og)

File: Circuit/digsig/adapter/msg_handler.js
const fs = require("fs")

const data = fs.readFileSync('response.json', 'utf8');
const obj = JSON.parse(data);
const { signature, ...json_msg } = obj;
const msg = Array.from(JSON.stringify(json_msg)).map(char => char.charCodeAt(0).toString());
const {r , s } = signature
const R = r.map(int => int.toString());
const S = s.toString()
module.exports = {R, S, msg}